tests/languages/plsql/operator_feature.test

```diff
@@ -1,9 +1,11 @@
 :=
+||
 
 ----------------------------------------------------
 
 [
-	["operator", ":="]
+	["operator", ":="],
+    ["operator", "||"]

```

Comment:
This is kind of nitpicky but could you please indent using tabs instead of spaces?